packages/manager/cypress/support/constants/widgets.ts

```diff
@@ -13,7 +13,7 @@ export const widgetDetails = {
     metrics: [
       {
         expectedAggregation: 'max',
-        expectedAggregationArray: ['avg', 'max', 'min', 'sum'],
```

Comment:
will there be no average?

Reply:
I have removed 'Avg' because the default value is already set to average. Changing the granularity to 'Avg' results in no network calls being fired, which is critical for automation as it relies on these network calls. Therefore, I've decided to remove 'Avg' to ensure proper functionality.